### ISSUE TITLE
Backbiter movement bug

### ISSUE THREAD

**Issue by comix1988** [azerothcore/azerothcore-wotlk#1379]
DESCRIPTION:

NPC Backbiter "dance" on site.

EXPECTED BLIZZLIKE BEHAVIOUR:

Backbiter is the warp stalker pet of Warp-Huntress Kula and follow her.

CURRENT BEHAVIOUR:

Backbiter "dance" on site. It does not follow Kula (https://wowgaming.altervista.org/aowow/?npc=32751)

STEPS TO REPRODUCE THE PROBLEM:

.go 5838.9 600.7 650.8 571

BRANCH(ES):
staging

AC HASH/COMMIT:
commit 987095e

OPERATING SYSTEM:
linux

MODULES:
OTHER CUSTOMIZATIONS:

**Comment by poszer** [comment]
Pet is stucked here? @comix1988 
https://user-images.githubusercontent.com/41213210/47912260-48a57b80-de98-11e8-88e2-8d94bb735bbd.png

**Comment by comix1988** [comment]
yep @poszer

**Comment by poszer** [comment]
> yep @poszer

Ok fix is referenced, try it

**Comment by comix1988** [comment]
@poszer edit:

today I came back from the pet and I found that it works properly until the dalaran round ends. In the ends backbiter "dancing" on site for a few seconds then come back to follow kula.

https://youtu.be/xzT_3A-Cods  ----> video link

Also if you use command .list creature 32751 the result is 0 and if you command .npc info while backbiter is selected the DBguid is 0.

![inkedwowscrnshot_020119_165908_li](https://user-images.githubusercontent.com/22867608/52134247-70e17980-2643-11e9-8b1c-7cc63140baef.jpg)
![inkedwowscrnshot_020119_164754_li](https://user-images.githubusercontent.com/22867608/52134248-70e17980-2643-11e9-86ff-711604b9586e.jpg)
